.github/workflows/build-cache.yml

```
@@ -0,0 +1,46 @@
+  pull_request:
+    branches: [main]
+
+env:
+  CI_ENVIRONMENT: 'true'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)